packages/@react-spectrum/avatar/docs/Avatar.mdx

```diff
@@ -0,0 +1,65 @@
+<!-- Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/avatar';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/avatar/package.json';
+
+```jsx import
+import {Avatar} from '@react-spectrum/avatar';
+```
+
+---
+category: Content
+keywords: [avatar]
+after_version: 3.0.0-alpha.0
+---
+
+# Avatar
+
+<p>{docs.exports.Avatar.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Avatar']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/avatar/'}
+  ]} />
+
+## Example
+
+```tsx example
+<Avatar alt="Zoe Saldana" src="https://i.imgur.com/jjiXMsdb.jpg" />
+```
+
+## Props
+
+<PropTable component={docs.exports.Avatar} links={docs.links} />
+
+## Visual options
+
+### Size
+
+```tsx example
+<Avatar alt="Zoe Saldana" size="size-1000" src="https://i.imgur.com/jjiXMsdb.jpg" />
```

Comment:
I don't think we can use her likeness in our docs, just a guess
we should use the same ones that spectrum.adobe.com has